controllers/operands/ssp.go

```diff
@@ -256,21 +257,31 @@ func getCommonDicts(list []hcov1beta1.DataImportCronTemplateStatus, crDicts map[
 		}
 
 		if crDict, found := crDicts[dictName]; found {
-			if !isDataImportCronTemplateEnabled(crDict) {
-				continue
-			}
-
 			// if the schedule is missing, copy from the common dict:
 			if len(crDict.Spec.Schedule) == 0 {
 				crDict.Spec.Schedule = targetDict.Spec.Schedule
 			}
 			targetDict.Spec = crDict.Spec.DeepCopy()
+
+			if targetDict.ObjectMeta.Annotations == nil && len(crDict.Annotations) > 0 {
+				targetDict.ObjectMeta.Annotations = map[string]string{}
+			}
+			maps.Copy(targetDict.ObjectMeta.Annotations, crDict.Annotations)
+
+			if targetDict.ObjectMeta.Labels == nil && len(crDict.Labels) > 0 {
+				targetDict.ObjectMeta.Labels = map[string]string{}
+			}
+			maps.Copy(targetDict.ObjectMeta.Labels, crDict.Labels)
+
 			targetDict.ObjectMeta.Namespace = crDict.Namespace
 			targetDict.Status.Modified = true
 		} else if ns := hc.Spec.CommonBootImageNamespace; ns != nil && len(*ns) > 0 {
 			targetDict.ObjectMeta.Namespace = *ns
 		}
 
+		if !isDataImportCronTemplateEnabled(targetDict.DataImportCronTemplate) {
+			continue
+		}
```

Comment:
If we say that we currently not supporting disabled common DICTs (from file), and I think we do, then we need to rollback these changes.

We only need to add filtering to the user defined DICTs.

Reply:
done